Documentation/Blazorise.Docs/Components/ComponentApiDocsProperty.razor

```diff
@@ -0,0 +1,60 @@
+﻿@using Blazorise.Generator.Features.ApiDocsDtos
+
+@if (filteredApiDocsForComponents.Any())
+{
+    <Heading Margin="Margin.Is2.FromTop.Is3.FromBottom" Size="HeadingSize.Is3">
+        @(PropertyType ==PropertyTypes.Events
+             ? "Events"
+            : "Parameters" )
+    </Heading>
+}
+
+@foreach(var apiDocsForComponent in filteredApiDocsForComponents)
+{
+    <DocsAttributes ShowDefaults="@(PropertyType != PropertyTypes.Events)"
+        Name="@(PropertyType == PropertyTypes.Events ? "Event": "Parameter")"
+        Title="@(MultipleComponentTypes ? apiDocsForComponent.TypeName : null)">
+        @foreach(var property in (PropertyType == PropertyTypes.Events? apiDocsForComponent.Events:apiDocsForComponent.Parameters).OrderBy(x => x.Name))
```

Comment:
Can we get some formatting in here? This almost looked like a null coalesce but it isn't right, it's a ternary operator, hard to read without proper spacing, maybe even replace with a getter propery if it's worth it.